src/main/java/com/techsophy/tsf/account/constants/AccountConstants.java

```diff
@@ -458,4 +458,21 @@
  public static final String MENU_ROLE_TABLE="tp_menu_role";
  public static final String MENU_ROLE="/menurole";
  public static final String MENU_ROLE_ID="/{id}";
+
+ //InternalUserConstants
+ public static final String USERNAME_INTERNAL ="userName";
+ public static final String FIRSTNAME_INTERNAL="firstName";
+ public static final String LASTNAME_INTERNAL="lastName";
+ public static final String MOBILE_NUMBER_INTERNAL="mobileNumber";
+ public static final String DEPARTMENT_INTERNAL="department";
+ public static final String SIGNATURE_INTERNAL="signature";
+ public static final String USERNAME_INTERNAL_VALUE="rupali.t@techsophy.com";
+ public static final String FIRSTNAME_INTERNAL_VALUE="rupali";
+ public static final String LASTNAME_INTERNAL_VALUE="tomar";
+ public static final String MOBILE_NUMBER_INTERNAL_VALUE="9893453678";
+ public static final String DEPARTMENT_INTERNAL_VALUE="IT";
+ public static final String INTERNAL="/internal";
+ public static final String USER_CREATE="/users/create/";
+ public static final String SIGNATURE_INTERNAL_VALUE="FBgH28Z9HajTQgWt1+UKa5kOvTcqQTH7rJq5ARV1QDZwq1gr0b703xMqh0BuDdPt1rFHGXr1j8BCI5886i2Q2lKif7bl087uc+p0AI542im8LKXfntvp9UDIi/PEkZnTsBFHm/2u0CTU6K6I2LxebIeBV5byZwi5re40b67MNH/xsoNKVUI55xXpZ2OH2JRj21qcTediu7KFdiTMEyNN6LMGPCDxT42eCPmf74bjEuctjDzKtksMcD8MX7q7G9ITvGdywnhWPzHskS4TzbKebjLpXQ43KZ4LEpauHOTYh/eNJZt1H45zUdGN7+kim3ctVLat0MQf0gxKMyean4httH6g9CQAtruNpbaMbklDao3SQmszi2acBdbgyi9EWZM5gNCOgWHSv+2Zwxza8yNqZTvQK1KlsupWduc9IMp6PpVUrLK8alCOfJ4TuEL/kBL7QooA7Bur4709lp3GLvp89P+zSW0NtvwM92o4flimPT8Xs9anprENCcvnHlYsPidOETqtAABBmTMftIzOto+Wsv2CK3KVzbwu9zRHQXzNrUKj9zvG/AmTygFqQ90LlL8gd/fS2BTP0Bi4OoYQFW2ypEX/5GhJDbrLydxuIMoOPSvUXH9tAMrpwXOXL+uEGaqnkKLm0lAflMl3zCHHZhAuau81GQhGi9JCEgDeIE6v2uM=";
```

Comment:
Dont see any reason why we need a constant for a signature

Reply:
for testcases I need one encrypted signature